cc3200/ftp/ftp.c

```diff
@@ -335,7 +335,7 @@ void ftp_run (void) {
                     ftp_data.loggin.uservalid = false;
                     ftp_data.loggin.passvalid = false;
                     strcpy (ftp_path, "/");
-                    ftp_send_reply (220, "Micropython FTP Server");
+                    ftp_send_reply (220, "MicroPython FTP Server");
```

Comment:
This is a (very mild) breaking change, because there may be some scripts out there that connect automatically via FTP and look for this particular string.  Eg cc3200/tools/update-wipy.py does automatic FTP, but luckily it doesn't check for this string.  So I'd say it's safe enough to make this change and brings good consistency.